optimum/onnxruntime/utils.py

```diff
@@ -271,3 +271,23 @@ class ORTQuantizableOperator(Enum):
     Resize = "Resize"
     AveragePool = "AveragePool"
     Concat = "Concat"
+
+
+def _get_external_data_paths(src_file_names, dst_file_names):
```

Comment:
Could you document (even just a sentence) this method? i think it makes it easier for people not super familiar with the code / ONNX / ORT to understand it.